src/main/java/org/candlepin/subscriptions/tally/job/CaptureSnapshotsTaskManager.java

```diff
@@ -86,42 +98,61 @@ public CaptureSnapshotsTaskManager(
    */
   @SuppressWarnings("indentation")
   public void updateAccountSnapshots(String accountNumber) {
+
+    String orgId = accountRepo.findOrgByAccountNumber(accountNumber);
+    if (StringUtils.hasText(orgId)) {
+      updateOrgSnapshots(orgId);
+    } else {
+      // Deprecate this method after org_id migration.
+      queue.enqueue(
+          TaskDescriptor.builder(TaskType.UPDATE_SNAPSHOTS, taskQueueProperties.getTopic())
+              .setSingleValuedArg("accounts", accountNumber)
+              .build());
+    }
```

Comment:
Given this shouldn't happen (we should always have record of a orgId we're about to tally), I'd rather throw an Exception here.

Reply:
Done.